src/library_gl.js

```diff
@@ -614,6 +614,17 @@ var LibraryGL = {
         }
       }
 
+      context.disjointTimerQueryExt = GLctx.getExtension("EXT_disjoint_timer_query");
+      if (context.disjointTimerQueryExt) {
+        GLctx['createQueryEXT'] = function() { return context.disjointTimerQueryExt['createQueryEXT'](); };
```

Comment:
I haven't tested if we can use non-timer related values with EXT versions of the API, not sure if that works? e.g. `ext.beginQuery(gl.ANY_SAMPLES_PASSED);` So you think we can replace regular versions with the EXT one and it should still work?